soiree/app/controllers/group/playlists_controller.rb

```diff
@@ -29,6 +29,21 @@ def delete_group_redirect(playlist)
     redirect_to dashboard_path
   end
 
+  def delete_group_invites(playlist)
+    #When a group member deletes their portion of the group playlist, their image should be taken off other group
```

Comment:
After playing around on my local server I've concluded that it doesn't, unfortunately.